content/docs/started/getting-started.md

```diff
@@ -82,7 +82,7 @@ kubectl create namespace ${NAMESPACE}
 # Which version of Kubeflow to use
 # For a list of releases refer to:
 # https://github.com/kubeflow/kubeflow/releases
-VERSION=v0.1.2
+VERSION=v0.2.0
 
 # Initialize a ksonnet app. Set the namespace for it's default environment.
 APP_NAME=my-kubeflow
```

Comment:
Thanks

Can you please remove the tf-job package and add the examples package?